src/core/public/styles/_ace_overrides.scss

```diff
@@ -12,6 +12,7 @@
 
     background-color: $euiColorLightestShade;
     color: $euiTextColor;
+    font-family: var(--font-code);
```

Comment:
Why is this a  `--font-code` and not `--font-text`?

Reply:
this is loaded by the code editor so we would like the code font.